### PR TITLE
Link mermaid docs to crate that uses them

### PR DESCRIPTION
**Pull request by Morganamilo**
While including files cross crate works locally. When pushed to crates.io each crate is seperate so the file path no longer exists.

Instead change it to a symlink, which cargo will change to a read directory when published.

### CODE REVIEW

File: substrate/primitives/runtime/docs [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../docs
```
Comment (bkchr) [inline substrate/primitives/runtime/docs:1]: I don't get what this file is doing?

Comment (ggwpez) [reply substrate/primitives/runtime/docs:1]: It think its GitHub's way of rendering links.

**Review by ggwpez — APPROVED** [review]
Yes this is why we need to run `cargo publish --dry-run` in the CI to check that all crates are in publishable state.

**Review by bkchr — APPROVED** [review]

**Comment by Morganamilo** [comment]
> Yes this is why we need to run `cargo publish --dry-run` in the CI to check that all crates are in publishable state.

It's not that simple as a dry run publish needs all of the dependencies published for real.

**Comment by ggwpez** [comment]
> It's not that simple as a dry run publish needs all of the dependencies published for real.

I remember JP forked and modified the code of the crates-io server to simulate that... but it was a pain to maintain.

**Pull request merged**